.github/workflows/docker_ci.yml

```diff
@@ -7,8 +7,8 @@ on:
 
 env:
   # Doesn't matter too much, we just want to verify the build passes
-  CURRENT_LTS: '0.22'
-  KB_VERSION: '0.22.26'
+  CURRENT_LTS: '0.23'
```

Comment:
Is this correct or do we need `0.24` here?

Reply:
Good catch, we need 0.24 here (and below).